src/compile.c

```diff
@@ -638,7 +639,7 @@ dcc_build_somewhere(char *argv[],
                                   needs_dotd ? deps_fname : NULL,
                                   server_stderr_fname,
                                   cpp_pid, local_cpu_lock_fd,
-                  host, status)) != 0) {
+                  host, status,hasgcov)) != 0) {
```

Comment:
Please add a space after the comma in lists.

Reply:
This lines has been resolved.